plone/app/locales/locales/it/LC_MESSAGES/plone.po

```diff
@@ -4227,7 +4227,7 @@ msgstr "Nessun URL o path inserito."
 
 #: CMFPlone/controlpanel/browser/redirects.py:302
 msgid "No alternative urls selected for removal."
-msgstr ""
+msgstr "Nessuna url alternativa selezionata."
```

Comment:
Nessun URL?

Reply:
+1 per il maschile.
Non ricordo dove sta questa voce e cosa fa, ma vista così mi chiedo perché in inglese specifica "for removal". Serve?